kolibri/plugins/management/assets/src/views/manage-content-page/wizards/import-preview.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+
+  <core-modal
+    :title="$tr('title')"
+    :enableBgClickCancel="false"
+    @cancel="cancel"
+    @enter="submit"
+    :enableBackBtn="true"
+    @back="goBack"
+  >
+    <div>
+
+      <!-- LOCAL IMPORT PREVIEW -->
+      <div v-if="importSource === 'local'">
+        <p>{{ localImportPrompt }}</p>
+
+        <ul class="channel-list">
+          <li v-for="(channel, i) in channelList" :key="i" >
```

Comment:
I guess so. I'm not so paranoid about the key being globally unique, though, since the list is pretty static.